backend/tests/benchmark_urgency_unoptimized.py

```
@@ -19,13 +19,13 @@
 def benchmark(iterations=10000):
     start_time = time.perf_counter()
     for _ in range(iterations):
-        priority_engine._calculate_urgency(sample_text, 10)
+        priority_engine.calculate_urgency(sample_text, 10)
     end_time = time.perf_counter()
 
     total_time = end_time - start_time
     avg_time_ms = (total_time / iterations) * 1000
 
-    print(f"Benchmark: _calculate_urgency")
+    print(f"Benchmark: calculate_urgency")
```

Ruff (0.15.14)

[error] f-string without any placeholders

Remove extraneous `f` prefix

(F541)